### PR TITLE
Dynamic menu options docs

### PR DESCRIPTION
**Pull request by jandonyada**

### CODE REVIEW

File: workbot/workbot-actions.md [hunk 1]
```diff
@@ -190,3 +190,45 @@ The fields available are simillar to that of the **Post command reply** action,
...
+
+![Grouped options](/assets/images/workbot/workbot-actions/grouped-options.png)
+*Grouped options in Slack*
+
+By default, you can specify a static list of menu options by adding menu options 1-by-1.
+
+![Add static menu option](/assets/images/workbot/workbot-actions/add-static-menu-option.gif)
+*Adding static menu options 1-by-1*
+
+However, you can create a dynamic list by changing the input mode to dynamic. Simply pass a list datapill from the ouput of a previous action step.
```
[the remark this answers is not in the document]
Comment (jandonyada) [reply workbot/workbot-actions.md]: Yea agreed. I want the the focus to be on dynamic lists so I'll prob do:
>Dynamic > Static > Groups
like in your last suggestion.

File: workbot/workbot-dynamic-menus.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+3. Returns the menu options to the primary command recipe.
+
+## Configuring a Workbot parameter with dynamic menu options
+In a Workbot command recipe,
+
+1. Create a parameter
+2. Under **Dialog control type**, choose **Select**.
+3. Under **Menu options**, choose **Dynamic**.
+4. Under dynamic menu recipe, enter the recipe ID of the dynamic menu recipe responsible for generating the menu options. At this point, you must have already created a dynamic menu recipe. To learn more about dynamic menu recipes, jump to [Dynamic menu recipes](#dynamic-menu-recipes).
+5. (Optional) If you want to send more context to the dynamic menu recipe, you can pass parameters to it using the **Dynamic menu recipe params** field.
```
[the remark this answers is not in the document]
Comment (jandonyada) [reply workbot/workbot-dynamic-menus.md]: Added a link to the trigger docs where this is explicitly described.

File: workbot/workbot-actions.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ Workbot supports 3 actions:
 * [Post notifications](#post-notifications)
```
Comment (bennettgo) [inline workbot/workbot-actions.md]: line 9 says 3 actions when it should be 5!

Comment (jandonyada) [reply workbot/workbot-actions.md]: Updated.

**Pull request merged**